edx_sga/tests/integration_tests.py

```diff
@@ -32,7 +32,7 @@
 from xblock.fields import ScopeIds
```

Comment:
Nit: please move the `StaffFactory` to the `common.djangoapps.student.tests.factories` import. The tests fail with an `ImportError` without this change.